apps/webbsite/src/components/shielded-pool/ShieldedPoolExtensionsSection.tsx

```diff
@@ -61,8 +61,8 @@ export const ShieldedPoolExtensionsSection = () => {
           className="md:text-[24px] md:leading-[40px] text-center text-mono-140"
         >
           We’re building out extensions to expand the capacity of shielded pool
-          protocols. Collaborate with us to help build extensions. To get started
-          building with them,
+          protocols. Collaborate with us to help build extensions. To get
+          started building with them,
```

Comment:
@devpavan04 can you fix this too, what's the copy for this supposed to be @thomivy/

"To get started building with them," sounds cut off from the full message?